torch/_inductor/mkldnn_lowerings.py

```diff
@@ -588,6 +588,14 @@ def qlinear_unary(
                     torch.tensor(x_scale, dtype=torch.float32), name="x_scale"
                 )
             else:
+                x_scale_size = x_scale.get_size()
+                if len(x_scale_size) == 2:
+                    if x_scale_size[0] == 1 and x_scale_size[1] == 1:
```

Comment:
```suggestion
            if all(i==1 for in x_scale.get_size()):
```

Reply:
Thank you! I applied this change at a couple of places.